packages/uni_app/lib/view/home/widgets/schedule_card.dart

```diff
@@ -49,6 +49,9 @@ class ScheduleCard extends GenericCard {
         ),
       ),
       contentLoadingWidget: const ScheduleCardShimmer().build(context),
+      mapper: (lectures) => lectures
+          .where((lecture) => lecture.endTime.compareTo(DateTime.now()) > 0)
```

Comment:
You can use isAfter here to make the code more readable.

Also, you should remove the code in getScheduleRows that does this again, as it is no longer needed!